astronomer/providers/amazon/aws/example_dags/example_emr_eks_containers_job.py

```diff
@@ -38,7 +38,7 @@
 AIRFLOW_HOME = os.getenv("AIRFLOW_HOME", "/usr/local/airflow")
 
 default_args = {
-    "execution_timeout": timedelta(minutes=30),
+    "execution_timeout": timedelta(hours=6),
```

Comment:
How did we come to this number of 6 hours?

Reply:
ok, doing it from UI is actually a good idea. I'll update the PR.
These DAG's run every 24hrs so anything less than that is ok but our execution time is 30-40 min so kept it 6hr.

Reply:
done